src/config/styles/html.js

```diff
@@ -6,22 +6,26 @@ import { imageHeight } from '../../helpers/imageHelper';
 
 export const html = {
   p: {
+    margin: 0,
```

Comment:
can you please sort all margins in alphabetically as the other entries

Reply:
https://github.com/smart-village-solutions/smart-village-app-app/commit/8705f45ee862bab3878eba4a848868d05380079c